README.md

```diff
@@ -3,7 +3,7 @@ kodus/uuid-v5
 
 Simple static UUID v5 generator/validator.
 
-[![PHP Version](https://img.shields.io/badge/php-7.0%2B-blue.svg)](https://packagist.org/packages/kodus/uuid-v4)
+[![PHP Version](https://img.shields.io/badge/php-8.0%2B-blue.svg)](https://packagist.org/packages/kodus/uuid-v4)
```

Comment:
Should this refer to uuid-v5?
Meaning https://packagist.org/packages/kodus/uuid-v5

Reply:
I'll get that changed 🤣